draft-ietf-masque-h3-datagram.md

```diff
@@ -241,7 +241,9 @@ negotiate the same draft version.
 ## HTTP Datagrams using Capsules
 
 When HTTP/3 Datagrams are unavailable or undesirable, HTTP Datagrams can be sent
-using the Capsule Protocol, see {{datagram-capsule}}.
+using the Capsule Protocol, see {{datagram-capsule}}. Capsules can also be used
+to communicate reliable and bidirectional control messages associated with a
```

Comment:
They're not necessarily associated with a datagram-based protocol

Reply:
I moved this down, but kept the language. The point is to clarify that although you're pointing out that you use it when you can't use QUIC datagrams, you still will want capsules for your datagram-based protocol. If you have other text in mind, please add it.

Reply:
Fair enough, this works for me